apps/redi-talent-pool/src/components/organisms/company-profile-editables/EditableJobPostings.tsx

```diff
@@ -56,27 +55,8 @@ export function EditableJobPostings({
     setIsEditing(true)
   }, [])
 
-  useEffect(() => {
-    if (isJobPostingFormOpen) {
-      setIsEditing(true)
-    }
-  }, [isJobPostingFormOpen])
-
-  useEffect(() => {
-    if (!isEditing) {
-      setIsJobPostingFormOpen(false)
-    }
-  }, [isEditing, setIsJobPostingFormOpen])
-
-  const handleStartEditingClick = (id, e: React.MouseEvent) => {
-    e.preventDefault()
-    startEditing(id)
-  }
-
-  const renderTimestamp = (expiresAt) =>
-    `Expires ${formatDistance(new Date(expiresAt), new Date(), {
-      addSuffix: true,
-    })}`
+  const hasJobListings = jobListings?.length > 0
+  const isEmpty = !hasJobListings
```

Comment:
Not sure if this is necessary. What do you think? (It seems like it only saves a few characters in expense of another memory allocation)

Reply:
Yeah, true, this is quite pointless and can easily be merged into one line. Doing that.